src/main/java/frc/robot/utils/MoPrefs.java

```diff
@@ -17,6 +17,9 @@ public final class MoPrefs {
     public static Pref<Double> driveCurve = doublePref("Drive Curve", 1);
     public static Pref<Double> driveSlowSpeed = doublePref("Drive Slow Speed", 0.5);
     public static Pref<Double> driveRampTime = doublePref("Drive Ramp Time", 0.25);
+    public static Pref<Double> shoulderSetpointRpm = doublePref("Shoulder Speed RPM", 200.0); // TODO: Find actual value
+    public static Pref<Double> wristSetpointRpm = doublePref("Wrist Speed RPM", 200.0); // TODO: Find actual value
+    public static Pref<Double> armJointSetpointRatio = doublePref("Wrist/Shoulder Speed Ratio", 1); // TODO: Find actual value
```

Comment:
What is the `armJointSetpointRatio`? I don't see it used anywhere?

Reply:
It is supposed to be used to keep the wrist parallel (maintain its angle relative to the ground) when raising: the ratio between the speed of the arm to the speed of the wrist